include/swift/AST/DiagnosticsParse.def

```diff
@@ -1524,6 +1524,8 @@ ERROR(expected_generics_type_restriction,none,
       (Identifier))
 ERROR(requires_single_equal,none,
       "use '==' for same-type requirements rather than '='", ())
+ERROR(requires_comma,none,
+      "use ',' for spliting conditions rather than '&&'", ())
```

Comment:
Maybe use `requirements` here? 
```suggestion
      "use ',' for spliting generic requirements rather than '&&'", ())
```

Reply:
As suggested by @CodaFi , I'll change this message to:

> expected ‘,’ to separate the requirements of this ‘where’ clause

Thanks.